cppclean

```diff
@@ -88,6 +88,17 @@ def main():
                         help='add a header include path; '
                              'specify this multiple times for multiple '
                              'include paths')
+    parser.add_argument('--include-path-system', '-is', '-Is', action='append',
```

Comment:
These short options seem a bit at odds with normal POSIX options, where `-is` would be equivalent to `-i` and `-s`.

Could you either rename it or remove it?